templates/dhcpd.conf.failover.erb

```diff
@@ -16,11 +16,10 @@ failover peer "dhcp-failover" {
 
 <%# This just moves the includes from dhcp.conf.erb to here so that any
 includes that involve failover pools won't fail. -%>
-<% includes = scope.lookupvar('dhcp::includes') -%>
```

Comment:
`@includes` isn't in scope for `dhcp::failover`, a scope lookup is required. See test failure with https://github.com/domcleal/puppet-dhcp/commit/8a8838f25e5016c3451a73822ff38fa312ad3c4d.